### PR TITLE
New fsq jet filtes - backport to 73

### PR DESCRIPTION
**Pull request by fruboes**
Backport https://github.com/cms-sw/cmssw/pull/7281

### CODE REVIEW

File: HLTrigger/JetMET/interface/HLTJetEtaTopologyFilter.h [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+   public:
+      explicit HLTJetEtaTopologyFilter(const edm::ParameterSet&);
+      ~HLTJetEtaTopologyFilter();
+      static void fillDescriptions(edm::ConfigurationDescriptions & descriptions);
+      virtual bool hltFilter(edm::Event&, const edm::EventSetup&, trigger::TriggerFilterObjectWithRefs & filterproduct) const override;
+
+   private:
+      const edm::InputTag inputJetTag_; // input tag identifying jets
+      edm::EDGetTokenT<std::vector<T>> m_theJetToken;
```
Comment (Martin-Grunewald) [inline HLTrigger/JetMET/interface/HLTJetEtaTopologyFilter.h]: Should also be const

**Comment by fruboes** [comment]
argh, sorry. I did  alle the needed changes in cc files, but forgot about one header. Anyway - fixed now in both PR

**Comment by Martin-Grunewald** [comment]
+1

**Comment by Martin-Grunewald** [comment]
@cmsbuild please test

**Comment by Martin-Grunewald** [comment]
@davidlange6
Please put this PR in 73X for the upcomming 73 release.
Thanks!

**Comment by davidlange6** [comment]
+1

**Pull request merged**